src/App.js

```diff
@@ -1,73 +1,30 @@
-import { useEffect, useState } from "react";
 import styled from "styled-components";
-import logo from "./logo.svg";
 
 function App() {
-  const [state, setState] = useState({ topic: "" });
-
-  useEffect(() => {
-    getData();
-
-    async function getData() {
-      const response = await fetch("/api/data-for-react");
-      const data = await response.json();
-      setState(data);
-    }
-  }, []);
   return (
     <Container>
-      <AppHeader>
-        <AppLogo src={logo} alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <AppLink
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Your React {state.topic} Template
-        </AppLink>
-      </AppHeader>
+      <StyledHeader>
+        <h1>Cook4All</h1>
+      </StyledHeader>
+      <p>Welcome!</p>
```

Comment:
This could be semantically a `h2` element.

Reply:
Right 👍